src/vivarium_nih_us_cvd/components/observers.py

```diff
@@ -292,74 +257,40 @@ def _get_categories(self) -> List[str]:
 
         return mapping[self.column]
 
-    def _get_population_view(self, builder: Builder) -> PopulationView:
-        columns_required = ["alive", self.column]
-        return builder.population.get_view(columns_required)
+    def calculate_categorical_person_time(self, x: pd.DataFrame) -> float:
+        return len(x) * to_years(self.step_size())
```

Comment:
This is the same aggregator function used by the `DiseaseObserver` to calculate person time. Is that function exposed in such a way that you can just call that? That would be preferable.

Reply:
I could import the DiseaseObserver and pass its aggregate_state_person_time method to the aggregator parameter. Would that suffice?

Reply:
It's not a static method, so I don't think that'll work. We should think about refactoring that slightly so that we have access to it.